cmd/minikube/cmd/start.go

```diff
@@ -408,11 +408,7 @@ func virtualBoxDeprecationWarning(driverName string) {
 		return
 	}
 	out.WarningT(`The 'virtualbox' driver is deprecated and will be removed in a future release.
-    You can use alternative drivers such as 'vfkit', 'qemu', or 'docker'.
-    https://minikube.sigs.k8s.io/docs/drivers/vfkit/
-    https://minikube.sigs.k8s.io/docs/drivers/qemu/
-    https://minikube.sigs.k8s.io/docs/drivers/docker/
-	`)
+    You can use alternative drivers: {{.drivers}}.`, out.V{"drivers": strings.Join(driver.SupportedDrivers(), ", ")})
```

Comment:
Why did you remove this? please keep the links to the docs. This make it very easy to find more info by clicking on the link in the shell, and it is not required for this change.